Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 - **Breaking Change** Renamed `ReactiveCocoaMoyaProvider` to `ReactiveSwiftMoyaProvider`.
 - Renamed the `ReactiveCocoa` subspec to `ReactiveSwift`.
+- `PluginType` can now modify requests and responses through `prepareRequest` and `processResponse`
```

Comment:
Since we're adding to the `PluginType` protocol, won't this be a breaking change?